test/network_autoconfiguration_test.rb

```diff
@@ -151,3 +151,27 @@ def probe_netcard_factory(num)
     expect(result["eth"]["eth1"]["DHCLIENT_SET_DEFAULT_ROUTE"]).to eq nil
   end
 end
+
+describe "NetworkAutoconfiguration#any_iface_active?" do
```

Comment:
This `describe` block should be inside the general `describe` block. I mean something like:

```ruby
describe Yast::NetworkAutoconfiguration do
   # already existing code

  describe "#any_iface_active?" do
    it "returns true..."
  end
end
```